### PR TITLE
(CGT-2528) Stubs for NRCGT Company

### PR DESCRIPTION
**Pull request by euptonqa**

### CODE REVIEW

**Comment by euptonqa** [comment]
Resolved merge conflicts

File: app/controllers/SubscriptionController.scala [hunk 1]
```diff
@@ -31,9 +31,10 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
-@Singleton
+
```
Comment (miranda-hawkes) [inline app/controllers/SubscriptionController.scala]: Please can you remove the unused import from removing this?

File: app/controllers/SubscriptionController.scala [hunk 2]
```diff
@@ -31,9 +31,10 @@ import scala.concurrent.ExecutionContext.Implicits.global
...
 class SubscriptionController @Inject()(actions: AuthorisedActions, registrationSubscriptionService: RegistrationSubscriptionService) extends BaseController {
 
+
```
Comment (miranda-hawkes) [inline app/controllers/SubscriptionController.scala]: Also just remove this extra line while you're at it please :)

File: app/controllers/SubscriptionController.scala (hunk 1 above)
Comment (euptonqa) [inline app/controllers/SubscriptionController.scala]: I don't think I was actually supposed to remove that Singleton, I'll put it back in if that's alright

**Pull request merged**